client/src/components/posting-page/JobPosting.js

```diff
@@ -78,8 +82,23 @@ export class JobPosting extends Component {
       return
     }
 
+    if (attachments.length > 0) {
+      promiseAttachments = await attachments.map((attachment) => {
```

Comment:
`await` keyword is not necessary here. `map` is not asynchronous and the promises are resolved on line 89